department-of-veterans-affairs/va.gov-cms
# VACMS-11736 Add perms for viewing sitewide alerts.

## Description

closes #11736

![image](https://user-images.githubusercontent.com/5752113/204698107-731a2223-3d42-47df-9f68-3ad2b5695c9b.png)

![image](https://user-images.githubusercontent.com/5752113/204698412-ac8ad48b-abb5-4d8b-83b0-ceb42a8b55d2.png)


## Testing done
## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

You will need two browsers to do this:

| Browser A | Incognito browser B |
|-----------------|------------------------------|
|[Login as ](https://pr11737-azmp4jgdgcnctsflptefdi5dfd4pkj2i.ci.cms.va.gov/)an admin | [login as](https://pr11737-azmp4jgdgcnctsflptefdi5dfd4pkj2i.ci.cms.va.gov/) `kitty.walsh` |
| [Go here](https://pr11737-azmp4jgdgcnctsflptefdi5dfd4pkj2i.ci.cms.va.gov/admin/content/sitewide_alert/add) to add a sitewide alert  |  |
|Name =Test Information | |
| Alert Style = Informational | |
| Alert message = Test message | |
|Save the sitewide alert | - [ ] Wait 15 seconds, then validate alert message displays just below the admin toolbar |
|Edit the alert, make it inactive then save | - [ ] Wait 15 seconds, then validate the alert no longer shows. |
|  [Go here](https://pr11737-azmp4jgdgcnctsflptefdi5dfd4pkj2i.ci.cms.va.gov/admin/content/sitewide_alert/add) to add a sitewide alert | |
|Name =Test Shield | |
| Alert Style = Full screen deployment shield | |
| Alert message = Test shield text | |
|Scheduling schedule the alert to end  ~2 minutes in the future||
|Save the sitewide alert | |
|- [ ] Wait 15 seconds, then validate shield alert covers all elements of the screen |- [ ] Wait 15 seconds, then validate shield alert covers all elements of the screen |
|- [ ] Wait ~2min and validate that the shield comes down.|- [ ] Wait ~2min and validate that he shield comes down.|
### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
